src/app/jscaip/tests/HexagonalGameState.spec.ts

```diff
@@ -185,10 +187,15 @@ describe('HexagonalGameState', () => {
             const line2: HexaLine = HexaLine.constantS(8);
             expect(state.getEntranceOnLine(line2).equals(new Coord(6, 2))).toBeTrue();
         });
-        it('should throw when unable to find an entrance', () => {
+        it('should call logError when unable to find an entrance', () => {
+            spyOn(ErrorLoggerService, 'logError').and.callFake(ErrorLoggerServiceMock.logError);
```

Comment:
GWT ME FREN

Reply:
done, ce genre de GWT ça fait un peu copie exacte du it par contre :D